Mobile/starter_project_matador/lib/features/example/presentation/screen/onboarding_1_screen.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:matador/features/example/presentation/widgets/animated_entry.dart';
+import 'package:matador/features/example/presentation/widgets/introduction.dart';
+
+class OnBoardingScreen1 extends StatefulWidget {
+  const OnBoardingScreen1({super.key});
+
+  @override
+  State<OnBoardingScreen1> createState() => _OnBoardingState();
+}
+
+class _OnBoardingState extends State<OnBoardingScreen1> {
+  double displaced = -350;
+  String topic = "Read the article you want instantly";
+  String content =
+      "You can read thousands of articles on Blog Club, save them in the application and share them with your loved ones.";
+  @override
+  void initState() {
+    super.initState();
+    Future.delayed(const Duration(milliseconds: 500), () {
+      setState(() {
+        displaced = 0;
+      });
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    var size = MediaQuery.of(context).size;
+    var longContainerWidth = size.width * (1 / 3) * 0.9;
+    var longContainerHeight = size.height * (3 / 5) * 0.4;
+    var wideContainerWidth = size.width * (2 / 3) * 0.9;
+    var wideContainerHeight = size.height * (3 / 5) * 0.4;
+    return Column(children: [
+      Container(
+        margin: EdgeInsets.only(top: size.height * 0.01),
+        padding: const EdgeInsets.all(10),
+        width: size.width,
+        height: size.height * (2.8 / 5),
+        child: Stack(
+          children: [
+            AnimatedEntry(
+              width: longContainerWidth,
+              height: longContainerHeight,
+              image: const AssetImage("assets/images/background1.png"),
+              top: displaced,
+              left: 0,
+            ),
+            AnimatedEntry(
+              width: wideContainerWidth,
+              height: wideContainerHeight,
+              image: const AssetImage("assets/images/background2.png"),
+              right: displaced,
+              top: 0,
+            ),
+            AnimatedEntry(
+              width: longContainerWidth,
+              height: longContainerHeight,
+              image: const AssetImage("assets/images/background3.png"),
+              bottom: displaced,
+              right: 0,
+            ),
+            AnimatedEntry(
+              width: wideContainerWidth,
+              height: wideContainerHeight,
+              image: const AssetImage("assets/images/background4.png"),
+              left: displaced,
+              bottom: 0,
+            ),
```

Comment:
if possible i would have this be in a loop

Reply:
As I said in this [PR](https://github.com/RealEskalate/Internship-2023/pull/16#discussion_r1174406831), doing it in a loop is not handy here. it is possible but leads to confussions and reduce code quality. This happens because dart and javascript differs in many ways. One thing is that, in dart it is not possible to create anonymous object as it is possible in javascript.

Reply:
I don't quite understand your point here. you can use different builders if needed.